libr/core/canal.c

```diff
@@ -540,7 +540,8 @@ static void r_anal_set_stringrefs(RCore *core, RAnalFunction *fcn) {
 }
 
 static bool r_anal_try_get_fcn(RCore *core, RAnalRef *ref, int fcndepth, int refdepth) {
-	ut16 bufsz = 1000;
+	ut16 bufsz = 1000; // XXX this is wrong
+	eprintf ("Warning: r_anal_try_get_fcn: is slow, should not use fixed bufsz\n");
```

Comment:
why do you say that it's slow because it allocates a big buffer?

Reply:
yes, but this function is almost never called, so its not really a problem for now, but i wont fix this in this pr

Reply:
use a R_LOG_DEBUG, it's more appropriate IMHO.